src/main/java/com/checkout/events/EventTypesResponse.java

```diff
@@ -1,16 +1,22 @@
 package com.checkout.events;
 
-import com.google.gson.annotations.SerializedName;
+import com.checkout.HttpMetadata;
+import com.checkout.ItemsResponse;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 
+import java.util.ArrayList;
 import java.util.List;
 
+
 @Data
-public final class EventTypesResponse {
+@EqualsAndHashCode(callSuper = true)
+public class EventTypesResponse extends HttpMetadata implements ItemsResponse<EventTypes> {
```

Comment:
Had a better idea for `items`. What do you think of something like this.

```
abstract class ItemsResponse<T> extends HttpMetadata {
    private List<T> items = new ArrayList<T>();
}

Then:
```
public final class EventTypesResponse extends ItemsResponse<EventTypes> {}
```

Ideally we could even always return a generic `ItemsResponse`, but I'm not sure if that would be possible.